### ISSUE TITLE
Trying to change downloads folder in HotApps module

### ISSUE THREAD

**Issue by shi-moro** [Jax-Core/ValliStart#48]
Heya.
My default downloads folder is changed (I have it on other disk), how can I change it in HotApps? I've tried to change path inside of `ValliStart\@Resources\Powershell\Func.ps1` file:

`Path="D:\Downloads"` instead of `Path="%USERNAME%\Downloads"`

 but it didn't really work - ValliStart still doesn't show any files in downloads folder and shows 0 as a size. What can I do?
![image](https://user-images.githubusercontent.com/61942497/198290025-e79f7a61-6f25-4fed-b7c0-626ebaa31084.png)

**Comment by EnhancedJax** [comment]
Change the path to the downloads folder in modules-> hotapps settings -> path. Make sure to undo any changes done to Vallistart manually.

**Comment by shi-moro** [comment]
> Change the path to the downloads folder in modules-> hotapps settings -> path.

Umm.. I feel stupid but I don't see any opportunity to change downloads directory there, that's why I tried to change it manually. I have only program list directory:

![image](https://user-images.githubusercontent.com/61942497/198550341-c2a19266-88a4-4627-8503-361360a35fcc.png)

**Comment by EnhancedJax** [comment]
Right. You'll have to edit line 114 of `Func.ps1` from `Path="%USERPROFILE%\Downloads"` to `Path="D:\Downloads"`. The change should apply after saving the file and reloading ValliStart. Make sure you've entered a correct path to the downloads folder.

**Comment by shi-moro** [comment]
Ye, I did it, and it doesn't work.
Also, I turned "List show folders" toggle. It still shows empty downloads folder, but when I choose `Up 1 directory`, it shows me disk D successfully. If I go to Downloads folder from it, it also shows it successfully:

<img src="https://user-images.githubusercontent.com/61942497/198556171-7faee198-9453-4609-a7a6-e795f7b6988d.png" width="350">
<img src="https://user-images.githubusercontent.com/61942497/198556208-c57c27c5-826a-438d-a6d0-a04725d3f990.png" width="350">
<img src="https://user-images.githubusercontent.com/61942497/198556252-0dae3bb7-aac4-4cec-8c91-a000733328de.png" width="350">
